docs/guide_running_portal_locally.md

```diff
@@ -0,0 +1,61 @@
+# Guide to running portal locally with a custom portal config
+
+This guide shows you how to test a portal config (aka `gitops.json` file) locally, so that you don't have to wait until the new `gitops.json` file is uploaded to GitHub and merged to see if your changes will break everything!
+
+Assuming you want to edit the `gitops.json` file for `preprod.healdata.org`:
+
+(Before you start make sure that `[preprod.healdata.org](http://preprod.healdata.org)` is up and running!)
```

Comment:
this link and 2 other links below are not working, probably because of the quotes